<launch>
    
    <!-- Choose if inertial or non-inertial model is desired -->
    <node name="car_simulation_node" pkg="sdv_control" type="car_simulation_node"/>
    <!-- <node name="car_simulation_node" pkg="sdv_control" type="inertial_car_simulation_node"/> -->

    <param name="robot_description" textfile="$(find sdv_control)/models/el_rasho.urdf" />

    <node name="tf2_broadcaster_node" pkg="sdv_control" type="tf2_broadcaster_node">
        <param name="parent_frame" value="world" />
        <param name="child_frame" value="sdv" />
    </node>

    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find sdv_control)/launch/rviz_cfg/sdv.rviz" />

</launch>